README.rst

```diff
@@ -102,7 +106,13 @@ using:
 
    user@host:~$ gpg --export <long_key_ID> > <long_key_ID>
 
-And then specify key ID to person mapping in the ``keyids`` file.
+And then specify the key ID to person mapping in the ``keyids`` file.
+
+Then please state in the ``git`` commit messages how confident you are that `no
+one has ever had a copy of your private key
+<https://www.wired.com/2014/10/snowdens-first-emails-to-poitras/>`_ and
+optionally, in case you feel like disclosing parts of your OpSec, the
+measures you implemented to prevent this from happening.
```

Comment:
I don't think that this paragraph needs to be here. Since current design of `debops-keyring` is based on `debian-keyring`, do Debian people require something like this to be publicly available, and stored forever in a signed `git` commit, to boot? I'm asking because I'm not up to speed on current Debian practices around this topic. I'll look for some info shortly.